### PR TITLE
Replace gitter by git discussions (#586)

### PR DESCRIPTION
**Pull request by alexvanboxel**
The gitter column is replaced by a discussion column to use a general term.
The Java community now points to git discussions instead of gitter.

### CODE REVIEW

File: README.md
Comment (yurishkuro) [inline README.md:114]:
```suggestion
Java: SDK|Every Friday at 08:00 PDT|[Google Doc](https://docs.google.com/document/d/1oHpisisco8J7Yk73ovx3eeXaokSMqgVuEuWNwPOajtY/edit)|[Zoom](https://zoom.us/j/8287234601)|[GitHub Discussions](https://github.com/open-telemetry/opentelemetry-java/discussions)|
```

**Review by yurishkuro — APPROVED** [review]

**Comment by Oberon00** [comment]
This should be approved by one of the @open-telemetry/java-maintainers.

File: README.md
Comment (anuraaga) [inline README.md:115]: This one too

https://github.com/open-telemetry/opentelemetry-java-instrumentation/discussions

**Review by anuraaga — COMMENTED** [review]
Thanks!

**Review by alolita — APPROVED** [review]
+1

File: README.md
Comment (alexvanboxel) [reply README.md]: Extra commit for replacing instrumentation as well.

**Review by anuraaga — APPROVED** [review]

**Comment by SergeyKanzhelev** [comment]
@anuraaga is gitter completely abandoned? Should we keep Gitter link as done in the Java repo?

image

**Comment by anuraaga** [comment]
@SergeyKanzhelev We were thinking of removing the gitter link from the README in the next release, but can remove it after this is merged if it reduce confusion.

**Pull request merged**
